src/devices/Shtc3/README.md

```diff
@@ -0,0 +1,26 @@
+# SHTC3 - Temperature & Humidity Sensor
+SHTC3 is a digital humidity and temperature sensor designed especially for battery-driven high-volume consumer electronics application.
+To reduce power cosumption this project use capability of sensor to allow measurement in low power mode and active sleep mode.
+
+## Usage
+```C#
```

Comment:
this actually sometimes misrenders if there is no extra enter between title and code block